README.md

```diff
@@ -69,9 +69,9 @@ Flame target, port 5300, TCP:
 flame -p 5300 -P tcp target.test.com
 ```
 
-Flame target, port 443, TCPTLS:
+Flame target, port 443, DoT (tcptls):
```

Comment:
I find _tcptls_ confusing. I suggest using _DNS-over-TLS_ or dropping the parens?

Reply:
`tcptls` is the old version right, so I assume that this is just to help people familiar with the old version?

Reply:
right I left it in for a bit of backwards compatibility, but happy to remove it.

Reply:
I think it's fine to leave in. I like the idea of not breaking scripts or the like.

Reply:
Personally, I would let the code accept the old option (which is already one) but remove from documentation (which is this comment). But it's cool if we keep it as it is. :+1: